spec/index.html

```diff
@@ -253,44 +279,59 @@ <h3>Example of CSV-Serialized Results</h3>
+        SPARQL [<a href="#SPARQL11-QUERY">SPARQL11-QUERY</a>] and Turtle [<cite><a class="bibref"
+        rel="biblioentry" href="#bib-TURTLE">TURTLE</a></cite>] use.</p>
+      <p>IRIs are enclosed in <code>&lt;...&gt;</code>, literals are enclosed with double quotes 
+        <code>"</code>...<code>"</code> or single quotes <code>'</code> ...<code>'</code> with optional
+        <code>@lang</code> or <code>^^</code> for datatype. IRIs are written enclosed in
+        <code>&lt;...&gt;</code>. They <em class="rfc2119" title="must">must</em> conform to the
+        <a href="https://www.rfc-editor.org/rfc/rfc3987#section-2.2">IRI rule</a> of [[[RFC3987]]].
+        Such IRIs include the IRI scheme and <em class="rfc2119" title="must not">must not</em> be a
+        <a href="https://www.rfc-editor.org/rfc/rfc3986#section-4.2">Relative Reference</a>. This
+        includes IRIs used as datatypes.
       </p>
-      <p>
-         Literals are written with the lexical form in quotes. Tab, newline and carriage return characters (Unicode codepoints 0x09, 0x0A (line feed) and 0x0D (Carriage Return)) are encoded in
-      strings as <code>\t</code>, <code>\n</code> and <code>\r</code> respectively. The long string forms using triple quotes <code>"""</code> and <code>'''</code> <em class="rfc2119" title="must not">must not</em> be
-      used. <!--  Emacs: " --></p>
-      <p>The abbreviated forms for numbers (XSD integers, decimals and doubles) <em class="rfc2119" title="should">should</em> be used.</p>
-      <p>Blank nodes use the <code>_:label</code> form from Turtle and SPARQL. Use of the same label indicates the same blank node within the results but has no significance outside the results.</p>
+      <p>Literals are written with the lexical form in quotes. Tab, newline, and carriage return characters
+        (Unicode codepoints <code>0x09</code> (tab), <code>0x0A</code> (line feed) and <code>0x0D</code>
+        (carriage return)) are encoded in strings as <code>\t</code>, <code>\n</code> and <code>\r</code>
+        respectively. The long string forms using triple quotes — <code>"""</code> or <code>'''</code> 
+        — <em class="rfc2119" title="must not">must not</em> be used. <!--  Emacs: " --></p>
+      <p>The abbreviated forms for numbers (XSD integers, decimals, and doubles) <em class="rfc2119"
+        title="should">should</em> be used.</p>
+      <p>Blank nodes use the <code>_:label</code> form from Turtle and SPARQL. Use of the same label
+        indicates the same blank node within the result set, but has no significance outside the result set.</p>
       </section>
 
       <section id="tsv-example">
       <h3>Example of TSV-Serialized Results</h3>
-      <p>Writing <code>&lt;TAB&gt;</code> for a raw tab character (Unicode code point 9):</p>
+      <p>Writing <code>&lt;TAB&gt;</code> for a raw tab character (Unicode code point <code>0x09</code>):</p>
```

Comment:
(Not for this PR)

Unicode is written `U+0009` and we seem to be converging on that across the documents.

Reply:
I see no reason not to apply this correction in this PR...
```suggestion
      <p>Writing <code>&lt;TAB&gt;</code> for a raw tab character (Unicode code point <code>U+0009</code>):</p>
```